File: JAVASCRIPT/Dinner/python/smallest.py
numbers = 67530
smallest = 0
number = 0
each_of_numbers =0
while number <=4:
	number+=1
	each_of_numbers = numbers%10
	numbers = (numbers-(numbers % 10))/10
	if each_of_numbers<smallest:
		smallest = each_of_numbers

print(smallest)

File: JAVASCRIPT/Dinner1/smallest.js
let numbers = 67530
let smallest = 0

for(let number = 0; number <=4; number++){
let eachOfNumbers = numbers%10
numbers = (numbers-(numbers % 10))/10
if (eachOfNumbers<smallest){
smallest = eachOfNumbers
}
}
console.log(smallest)
